lib/server/services/log/LogService.js

```diff
@@ -131,17 +145,36 @@ class LogService {
      * @return {Promise<Log[]>} the resulting logs
      */
     async _getAllByIds(logIds) {
-        const logQueryBuilder = dataSource.createQueryBuilder()
-            .where('id').oneOf(...logIds)
-            .include({ association: 'runs', attributes: ['id', 'runNumber'] })
-            .include({ association: 'lhcFills', attributes: ['fillNumber'] })
-            .include({ association: 'environments', attributes: ['id'] });
+        const logQueryBuilder = this._createFindQueryBuilder();
 
+        logQueryBuilder.where('id').oneOf(...logIds);
         const logs = await LogRepository.findAll(logQueryBuilder);
+        return this._prepareLogsEntities(logs);
+    }
+
+    /**
+     * Return all the logs after preparing them to entities and adding childrenCount
+     *
+     * @param {logs[]} logs the list of logs to fetch
```

Comment:
logs to fetch? Rather to prepare no?
The type is `SequelizeLog[]` not `logs[]`

Reply:
done